src/hotspot/share/cds/filemap.cpp

```diff
@@ -2018,44 +2018,44 @@ bool FileMapInfo::map_heap_data(MemRegion **heap_mem, int first,
 
     if (VerifySharedSpaces && !region_crc_check(addr, regions[i].byte_size(), si->crc())) {
       // dealloc the regions from java heap
-      dealloc_archive_heap_regions(regions, region_num);
+      dealloc_heap_regions(regions, region_num);
       log_info(cds)("UseSharedSpaces: mapped heap regions are corrupt");
       return false;
     }
   }
 
   cleanup._aborted = false;
   // the shared heap data is mapped successfully
-  *heap_mem = regions;
-  *num = region_num;
+  *regions_ret = regions;
+  *num_ret = region_num;
```

Comment:
Instead of `num_ret`, I `num_regions` is easier to understand.

Reply:
I renamed it to `num_regions_ret` to be consistent with `regions_ret`: both of them are outgoing parameters. Now their names are also consistent with the globals like `closed_heap_regions` and `num_closed_heap_regions`.

I also rearranged the parameter order to put the outgoing ones at the end.